pkg/scheduler/metrics/metrics.go

```diff
@@ -271,6 +271,13 @@ func Register() {
 	})
 }
 
+// RegisterExtraMetrics registers extra metrics.
+func RegisterExtraMetrics(extraMetrics ...metrics.Registerable) {
```

Comment:
```suggestion
func RegisterMetrics(extraMetrics ...metrics.Registerable) {
```

Reply:
Use this function inplace of lines 264-266

Reply:
Done, thanks

Reply:
thanks, done